### PR TITLE
Update kwallet ssh integration

### PR DESCRIPTION
**Pull request by Girtablulu**
## Description

<Info on what this pull request adds>

### Submitter Checklist

- [x] Updated the "lastmod" portion at the top of any modified Markdown files.
- [x] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: software/kwallet/en.md [hunk 1]
```diff
@@ -31,14 +31,33 @@ The contents of `~/.config/autostart-scripts/ssh-add.sh` should reflect the SSH
 Below is an example of the contents of `.config/autostart-scripts/ssh-add.sh`:
 
 ```
-#!/bin/sh
+#!/bin/bash
 
-ssh-add $HOME/.ssh/id_rsa $HOME/.ssh/solus_id_rsa </dev/null
+ssh-add ssh-add -q < /dev/null
```
Comment (JoshStrobl) [inline software/kwallet/en.md:36]: Duplicate ssh-add

File: software/kwallet/en.md [hunk 2]
```diff
@@ -31,14 +31,33 @@ The contents of `~/.config/autostart-scripts/ssh-add.sh` should reflect the SSH
 Below is an example of the contents of `.config/autostart-scripts/ssh-add.sh`:
 
 ```
-#!/bin/sh
+#!/bin/bash
```
Comment (JoshStrobl) [inline software/kwallet/en.md:34]: Why are we forcing bash now?

File: software/kwallet/en.md [hunk 3]
```diff
@@ -31,14 +31,33 @@ The contents of `~/.config/autostart-scripts/ssh-add.sh` should reflect the SSH
...
-ssh-add $HOME/.ssh/id_rsa $HOME/.ssh/solus_id_rsa </dev/null
+ssh-add ssh-add -q < /dev/null
 ```
 
+Tip: The above ssh-add.sh script will only add the default key `~/.ssh/id_rsa`. Assuming you have different keys named key1, key2 etc you need to change the above script to
+
+```
+#!/bin/bash
+
+ssh-add ssh-add ~/.ssh/key1  ~/.ssh/key2 < /dev/null
```
Comment (JoshStrobl) [inline software/kwallet/en.md:44]: Duplicate ssh-add

File: software/kwallet/en.md [hunk 4]
```diff
@@ -31,14 +31,33 @@ The contents of `~/.config/autostart-scripts/ssh-add.sh` should reflect the SSH
...
+#!/bin/bash
+
+export SSH_ASKPASS='/usr/bin/ksshaskpass'
+```
+
+
 After creating this file, you will need to make it executable:
 
-`chmod a+x ~/.config/autostart-scripts/ssh-add.sh`
+`chmod +x ~/.config/plasma-workspace/env/askpass.sh`
```
Comment (JoshStrobl) [inline software/kwallet/en.md:60]: Why was this changed? a+ ensures the rest of the modes are not affected

**Review by JoshStrobl — CHANGES_REQUESTED** [review]

File: software/kwallet/en.md (hunk 4 above)
Comment (Girtablulu) [reply software/kwallet/en.md]: they guide I read was saying just make it executable that's why I changed it

Comment (JoshStrobl) [reply software/kwallet/en.md]: That can be intepreted as "the only action you need to do is make it +x", thus `a+x` would still be accurate.

**Comment by JoshStrobl** [comment]
LGTM, thanks!
